### ISSUE TITLE
Does the `noselect` option (lack of it in `completeopts`) work with `longest`?

### ISSUE THREAD

**Issue by psprint** [lifepillar/vim-mucomplete#183]
Hi,
I would like to have the first completion always selected – immediately after appearing. As far as I understood, this should be achieved via:

```vim
set completeopt-=noselect
" BTW, the other options that I set are:
set completeopt+=menuone,noinsert,longest
```

However, this isn't the case, unless I also remove the `longest` option. Could someone explain me why?

**Comment by lifepillar** [comment]
From `:help cot`:

> 	   noinsert Do not insert any text for a match until the user selects
>		    a match from the menu. Only works in combination with
>		    "menu" or "menuone". No effect if "longest" is present.

**Comment by lifepillar** [comment]
Do you want to insert the longest common text of the matches *and* have the first menu entry selected? I may be wrong, but I don't think that Vim supports that.

**Comment by psprint** [comment]
Yes, this is what I want. I've created a [ticket](https://github.com/vim/vim/issues/6463).
